src/Optimizers/KaHyPar.jl

```diff
@@ -7,6 +7,8 @@ using KaHyPar
     imbalance::Float32 = 0.03
     stop::Function = <=(2) ∘ length ∘ Base.Fix1(getfield, :args)
     configuration::Union{Nothing,Symbol,String} = nothing
+    edge_scaler::Function = (ind_size) -> 1000 * (Int ∘ round ∘ log2)(ind_size)
+    vertex_scaler::Function = (prod_size) -> 1000 * (Int ∘ round ∘ log2)(prod_size)
```

Comment:
Just putting some `∘` but keeping the `->` does not remove the closure: it will still be a closure.

```suggestion
    edge_scaler::Function = *(1000) ∘ Int ∘ round ∘ log2
    vertex_scaler::Function = *(1000) ∘ Int ∘ round ∘ log2
```

Reply:
Okey!